<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WLW7-KavithaLN" name="Test A2" tests="3" failures="1" timestamp="17 Jul 2014 13:06:32 GMT" time="0.002" errors="0">
  <testcase name="testA2_1" time="0.0" classname="testSuiteA.TestA2">
    <skipped/>
  </testcase> <!-- testA2_1 -->
  <testcase name="testA2_2" time="0.0" classname="testSuiteA.TestA2"/>
  <testcase name="testA2_3" time="0.0" classname="testSuiteA.TestA2">
    <failure type="java.lang.AssertionError" message="failed test case">
      <![CDATA[java.lang.AssertionError: failed test case
	at testSuiteA.TestA2.testA2_3(TestA2.java:24)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testA2_3 -->
</testsuite> <!-- Test A2 -->
